components/Blog.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import BlogCard from './BlogCard';
+import BlogData from '../data/blog';
+
+const Blog = () => {
+  return (
+    <section className="px-4 py-16 mx-auto sm:max-w-xl md:max-w-full lg:max-w-screen-xl md:px-24 lg:px-8 lg:py-20">
+      <div className="max-w-xl mb-10 md:mx-auto sm:text-center lg:max-w-2xl md:mb-12">
+        <h2 className="max-w-lg mb-6 font-sans text-3xl font-bold leading-none tracking-tight text-gray-900 sm:text-4xl md:mx-auto">
+          <span className="relative">Articles & Guide</span>
+        </h2>
+        <p className="text-base text-gray-700 md:text-lg">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elementum et accumsan
+        </p>
+      </div>
+      <div className="grid gap-6 row-gap-5 mb-8 lg:grid-cols-3 sm:row-gap-6 sm:grid-cols-2">
+        <>
+          {BlogData.map((data, index) => {
+            return <BlogCard key={index} article={data.article} src={data.image} press_release={data.press_release} />;
```

Comment:
We are currently using camelCase for props in Javascript, so you don't need to put underscore in the prop name.

```suggestion
            return <BlogCard key={index} article={data.article} src={data.image} pressRelease={data.press_release} />;
```